### PR TITLE
Added the ability to reset the recorder

### PR DESCRIPTION
**Pull request by grellyd**
Resets the start time and the recorded request collection.

### CODE REVIEW

File: recorder.go [hunk 1]
```diff
@@ -38,6 +45,11 @@ func (rr *RequestRecorder) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if r.URL.Path == "/gopherman-reset" {
```
Comment (cohix) [inline recorder.go:48]: Let's just reset the collection in the terminate handler, rather than having it be a separate handler.

Comment (grellyd) [reply recorder.go:48]: I'd prefer to have them separate. I've found it to be handy to be able to check on the requests, as you are building up to a more complex tasks. 

If you want terminate to include reset, then how about adding a second endpoint that does not reset, like `gopherman-current`?

Comment (grellyd) [reply recorder.go:48]: Aka swap the endpoint roles.

Comment (cohix) [reply recorder.go:48]: Alright I get your reasoning. This can stay as is.

**Review by cohix — APPROVED** [review]

**Pull request merged**